strapi/strapi
# List view: new cog button icon with the view settings in the list view page

### What does it do?

On the List View page, we would like to free up some space above the table, in the top right corner, to show only a Cog Icon that can open a menu with a link to the List View settings and checkboxes for selecting the columns to display in the List View.
Additionally, we want to add a reset button that selects the default columns of the collection.

### Why is it needed?

In the list view, there are 2 elements in the top right corner of the table:

- The columns that are currently selected
- The cog icon that redirects to the settings of the list view (Configure the view)

We’d like to merge them to free some space in the list view.

### How to test it?

 - Go the content manager and select a collection
 - On the list view you can see a Cog icon on the top right corner of the table
   - click the cog icon
     - you can click on the "Configure the view" button to go to the list view settings page of the collection
     - you can select/deselect the headers to show in the list view
     - you can click on the reset button and immediately have the default selection of the headers for the collectios

### Related issue(s)/PR(s)

The ticket related is CS-42
